contrib/log_correlation/stackdriver/src/test/java/io/opencensus/contrib/logcorrelation/stackdriver/OpenCensusTraceLoggingEnhancerTest.java

```diff
@@ -105,6 +177,86 @@ private static LogEntry getEnhancedLogEntry(String projectId, Span span) {
     }
   }
 
+  private static void assertContainsNoTracingData(LogEntry logEntry) {
+    assertThat(logEntry.getLabels()).doesNotContainKey("sampled");
+    assertThat(logEntry.getTrace()).isNull();
+    assertThat(logEntry.getSpanId()).isNull();
+  }
+
+  @Test
+  public void spanSelectionDefaultIsSampledSpans() {
+    assertThat(new OpenCensusTraceLoggingEnhancer().getSpanSelection())
+        .isEqualTo(SpanSelection.SAMPLED_SPANS);
+  }
+
+  @Test
+  @SuppressWarnings("TruthConstantAsserts")
+  public void spanSelectionPropertyName() {
+    assertThat(OpenCensusTraceLoggingEnhancer.SPAN_SELECTION_PROPERTY_NAME)
```

Comment:
Why do we need to test this?

Reply:
Makes sure the class name matches up. :)

Reply:
Yes, I had to use a String literal in the non-test code so that I could display it in the Javadoc, and I wanted to check that it matched the class name.  This is also important to test because users will need to specify it in their logging configurations.

Reply:
why not initialize using class.getName().

```
public static final String SPAN_SELECTION_PROPERTY_NAME =
  OpenCensusTraceLoggingEnhancer.class.getName() + ".spanSelection";
```

Reply:
I wanted to display the value with a `@value` tag in the Javadoc, so I had to use a String literal.

Reply:
got it. thx.